mantidproject/mantid
# Introduction to python docs converted to Sphinx

**Description of work.**
This PR continues the conversion of docs from media wiki to Sphinx. In this instance the `Introduction to Python` has been copied over.  

**To test:**

* Build the `docs-html` target.
* Open `html/index.html`
* Navigate to `Training` then `Introduction to Python`
* Check that the pages have built and links work
* Test some of the examples to see if they work
 
*There is no associated issue.*



*This does not require release notes* because it is not a change to the code.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
